snapshots.js

```diff
@@ -1,8 +1,5 @@
-// Copyright 2022-2024 use-ink/contracts-ui authors & contributors
-// SPDX-License-Identifier: GPL-3.0-only
-
```

Comment:
Why is this removed?

Reply:
Thanks! Auto-generated file, will get the license back in